packages/skill-api/src/core/lib/convertkit.ts

```diff
@@ -0,0 +1,77 @@
+import {isEmpty} from 'lodash'
+
+const convertkitBaseUrl =
+  process.env.CONVERTKIT_BASE_URL || 'https://api.convertkit.com/v3/'
+
+export async function setConvertkitSubscriberFields(
```

Comment:
did these over here without axios because I was trying to figure out the mjml thing, but liked them